src/QueueReceiver.Infrastructure/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.EntityFrameworkCore;
+using QueueReceiver.Core.Models;
+
+namespace QueueReceiver.Infrastructure.Data
+{
+    public class ApplicationDbContext : DbContext
+    {
+        public ApplicationDbContext(
+            DbContextOptions<ApplicationDbContext> options
+        ) : base(options)
+        {
+        }
+
+        public virtual DbSet<Person> Persons { get; set; } = null!;
+        public virtual DbSet<PersonProject> Personprojects { get; set; } = null!;
+        public virtual DbSet<Project> Projects { get; set; } = null!;
+        public virtual DbSet<Plant> Plants { get; set; } = null!;
+        public virtual DbSet<PersonUserGroup> PersonUserGroups { get; set; } = null!;
+        public virtual DbSet<PersonProjectHistory> PersonProjectHistories { get; set; } = null!;
+        public virtual DbSet<UserGroup> UserGroups { get; set; } = null!;
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<PersonProject>()
```

Comment:
Consider moving to separate configuration classes, one for each model (google it). Perhaps a bit overkill with the current complexity, but it's nice and clean :)